mage/query-modules/python/meta-util.md

```diff
@@ -0,0 +1,398 @@
+---
+title: meta_util
+sidebar_label: meta_util
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+  <span
+    style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+    {children}
+  </span>
+);
+
+<!-- TODO -- update link -->
+[![docs-source](https://img.shields.io/badge/source-meta_util-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/meta_util.py)
+
+
+## Abstract
+
+A module that contains procedures describing graphs on a meta-level.
+
+| Trait               | Value                                                 |
+| ------------------- | ----------------------------------------------------- |
+| **Module type**     | <Highlight color="#FB6E00">**util**</Highlight>  |
+| **Implementation**  | <Highlight color="#FB6E00">**Python**</Highlight>     |
+| **Parallelism**     | <Highlight color="#FB6E00">**sequential**</Highlight> |
+
+## Procedures
+
+<RunOnSubgraph/>
+
+### `schema(include_properties)`
+
+Returns a list of distict relationships connecting distinct nodes, that is, a graph schema. If `include_properties` is set to `true`, the graph schema will contain properties count information. 
+
+#### Input:
+
+* `include_properties: bool (default=false)` ➡ If set to `true`, the graph schema will include properties count information. 
```

Comment:
See above
properties count doesn't mean much if you don't know what it's really about?

Reply:
I see know you use it quite a lot throughout the documentation... but I'm still wondering if people will know what it is, and what you mean?

Reply:
I added a sentence at the beginning of the procedure description. Let me know if that clears things up a bit or should I go fix it all?